data/campaigns/Descent_Into_Darkness/scenarios/05_Orc_War.cfg

```diff
@@ -317,22 +362,22 @@
 
         [message]
             speaker=Malin Keshar
-            message= _ "The last chieftain... he’s escaping! The coward!"
+            message= _ "The last chieftain flees! The coward!"
         [/message]
 
         [message]
             speaker=Darken Volk
-            message= _ "Coward, yes. You must hunt him down before he can take word to the orcish council. A single chieftain, even after he gathers the survivors from this battle, should pose no problem for you, even on your own."
+            message= _ "Orcs are no more than beasts. They bellow out against a weaker foe as if that makes them mighty, but will always flee in the face of an overwhelming force. Nevertheless, the Whitefangs still number many and have some ties to the other orc clans as well. You must hunt this chieftain down before he can raise more reinforcements, else we may soon have dozens more on our tail."
```

Comment:
This feels like racist propaganda, as did some of the dialogue in the earlier PRs. It's logical given what Darken is trying to do to Malin, but for a game I suggest this should be a case of Wesnoth Is Not Realistic.

Reply:
I feel that Wesnoth as a whole shouldn't push that "orcs are beasts" (and personally I would really like another orc campaign). However, I think DiD is the campaign that it makes the least sense to do this in. The whole campaign is based around Malin's hatred for the orcs, and as you say, it only makes sense for Darken Volk to prey on this to manipulate him. In S7-9, we'll see that Darken Volk is just as disparaging to humans; he pretty much doesn't think very much of anyone besides himself. I don't think Malin should every try to be anything less than completely hateful of orcs.

... I still actually don't like the idea of pushing that orcs are beasts too much; I was hoping to write a bit of a scenario where we might even pity them as well or at least get their viewpoint on the necromancers. It could very well be this scenario but I wanted to avoid changing it toooo much.

Reply:
I retract my comment - in the context of the campaign it fits, and it feels balanced by the unbiased between-scenario narrator.